projects/demo/src/modules/components/tooltip/examples/1/index.less

```diff
@@ -4,3 +4,15 @@
     height: 1.25rem;
     min-width: 6.25rem;
 }
+
+.wrapping-tooltip {
+    width: fit-content;
```

Comment:
How about we add:
```css
resize: horizontal;
overflow: auto;
```
so we can play around with it without resizing the whole window?

Reply:
Great idea! I'll do it :)